docs/conf.py

```diff
@@ -9,7 +9,7 @@
 project = "celi-framework"
 copyright = "2024, Jan-Samuel Wagner"
 author = "Jan-Samuel Wagner"
-release = "0.0.0"
+release = "0.0.11"
```

Comment:
This should stay as 0.0.0 in the main branch.  When the release package is created it gets replaced with the right value.  It happens in the ` poetry version` call in .github\workflows\publish.yml